src/geo/cartodb-layer-group.js

```diff
@@ -18,24 +19,24 @@ var CartoDBLayerGroup = Backbone.Model.extend({
     this._layersCollection = options.layersCollection;
   },
 
-  contains: function (layerModel) {
-    return this.getIndexOf(layerModel) >= 0;
+  forEachGroupedLayer: function (iteratee, context) {
```

Comment:
CartoDB layers are being grouped by this CartoDBLayerGroup, so I figured "grouped layers" would be a good name for those.